generator/src/main/java/com/scottlogic/deg/generator/generation/combinationstrategies/ReductiveCombinationStrategy.java

```diff
@@ -16,34 +16,30 @@
 
 package com.scottlogic.deg.generator.generation.combinationstrategies;
 
-import com.scottlogic.deg.common.util.FlatMappingSpliterator;
 import com.scottlogic.deg.generator.generation.databags.DataBag;
-import com.scottlogic.deg.generator.utils.RestartableIterator;
 
 import java.util.List;
-import java.util.Spliterator;
-import java.util.Spliterators;
+import java.util.function.Supplier;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import java.util.stream.StreamSupport;
+
+import static com.scottlogic.deg.common.util.FlatMappingSpliterator.flatMap;
```

Comment:
A side note (I don't expect it to be addressed in this PR): I don't think we should have package names including `com.scottlogic` now the project is in FinOS.